internal/caddy/config.go

```diff
@@ -71,3 +70,10 @@ func NewConfig(namespace string, cfg ControllerConfig) *Config {
 		},
 	}
 }
+
+func getCAEndpoint(useStaging bool) string {
+	if useStaging {
+		return "https://acme-staging-v02.api.letsencrypt.org/directory"
+	}
+	return ""
```

Comment:
This should not be the "official" letsencrypth endpoint?

Reply:
Also, do you think we can avoid hard-coding any CA endpoints? Caddy will already use Let's Encrypt by default, for public certificates.

Reply:
> This should not be the "official" letsencrypth endpoint?

I kept it empty to use the caddy default but I can set it to a "static" endpoint.

> Also, do you think we can avoid hard-coding any CA endpoints?

Yep, I think it'd be better. With the ConfigMap system, it wouldn't be a flag but would be in the ConfigMap.

I reused the existing flag but I can remove it

Reply:
@Embraser01 @mholt I'll like to merge this PR and than refactor this part in order to use a ConfigMap to configura Caddy ACME, WDYT?

Reply:
Yep I :+1:
I will open a draft pr with my work on config map (with already the acme url available)